src/commons/body-message-renderer.jsx

```diff
@@ -3,22 +3,35 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-import React, { useCallback, useEffect, useRef } from 'react';
-import { forEach, reduce } from 'lodash';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+import { forEach, reduce, map } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { Container, Text } from '@zextras/carbonio-design-system';
 
 const _CI_REGEX = /^<(.*)>$/;
 const _CI_SRC_REGEX = /^cid:(.*)$/;
+const LINK_REGEX = /\bhttps?:\/\/\S+/g;
 
-function TextMessageRenderer({ text }) {
-	const containerRef = useRef();
-
-	useEffect(() => {
-		containerRef.current.innerText = text;
-	}, [text]);
+export const locationUrl = (location) => {
+	const found = location?.match(LINK_REGEX);
+	return found?.length ? found[0] : undefined;
+};
 
-	return <Text ref={containerRef} />;
+function TextMessageRenderer({ text }) {
```

Comment:
in this case only the urls inside the text are replaced with links, iirc the requirement was to do it in the appt location, right?